README.md

```diff
@@ -22,6 +22,14 @@ ddf = dask_bigquery.read_gbq(
 ddf.head()
 ```
 
+## Run tests locally
+
+To run the tests locally you need to be authenticated and have a project created on that account. If you're using a service account, when created you need to select the role of "BigQuery Admin" in the section "Grant this service account access to project". 
+
+In the directory `dask_bigquery/tests` you can run the tests by doing:
+
+`DASK_BIGQUERY_PROJECT_ID=your_project_id pytest test_core.py`
```

Comment:
Let's suggest folks run tests from the root of the `dask-bigquery` repo. This is less important for this project as the `tests/` directory doesn't have any nested structure, but it's usually common to run tests from the root of the repo to avoid changing directories. I suspect `pytest dask_bigquery` will be more familiar to developers than `cd dask_bigquery/tests; pytest test_core.py`. For example, something like: 

--- 

Test can be run from the root of the `dask-bigquery` repository with:

```bash
$ DASK_BIGQUERY_PROJECT_ID=<your_project_id> pytest dask_bigquery
```

Reply:
Yes you are right, will modify that! Thanks for the comment